src/plays/drawing-board/store.ts

```diff
@@ -0,0 +1,17 @@
+import { configureStore } from "@reduxjs/toolkit";
+import { logger } from "redux-logger";
+import historyIndex from "./modules/historyIndex/slice";
+import { currentStroke } from "./modules/currentStroke/slice";
+import strokes from "./modules/strokes/slice";
+import { modalVisible } from "./modules/modals/slice";
+
+export const store = configureStore({
+  reducer: {
+    historyIndex,
+    currentStroke,
+    strokes,
+    modalVisible,
+  },
+
+  middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),
+});
```

Comment:
this logger is spaming console with thousands of console log. i think we remove this.

Reply:
Done